lib/cadet_web/views/assessments_view.ex

```diff
@@ -141,7 +141,7 @@ defmodule CadetWeb.AssessmentsView do
     end
   end
 
-  defp build_result(result) do
+  def build_result(result) do
```

Comment:
I think this belongs in view_helpers now since its being shared?

Reply:
It should still be fine leaving it under assessments_view? At most only grading_view and assessments_view require build_result so I don't think it makes sense to isolate the function.

Reply:
the pattern we have been adopting is all shared methods go into the view helpers so our views dont couple or depend on other views!! lets try to be consistent with the codebase okay!!